<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signup_Dialog</class>
 <widget class="QDialog" name="signup_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_signup">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>103</y>
     <width>531</width>
     <height>431</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="lineEdit_user">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_pass">
    <property name="geometry">
     <rect>
      <x>181</x>
      <y>260</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_confpass">
    <property name="geometry">
     <rect>
      <x>181</x>
      <y>300</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sign">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>360</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Sign Up</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_phonenum">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>220</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>SIGN UP AS JOBSEEKER</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_signpic">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>31</y>
      <width>161</width>
      <height>381</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cancel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>360</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_address">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_gmail">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>140</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_second_status">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
